src/dioptra/client/_client.py

```diff
@@ -0,0 +1,707 @@
+        return cast(dict[str, Any], requests.get(queue_name_query).json())
+
+    def get_builtin_task_plugin(self, name: str) -> dict[str, Any]:
+        """Gets a custom builtin plugin by its unique name.
+
+        Args:
+            name: A unique string identifying a task plugin package within dioptra_builtins collection.
+
+        Returns:
+            The Dioptra REST api's response.
+
+            Example::
+
+                {
+                    'taskPluginName': 'attacks',
+                    'collection': 'dioptra_builtins',
+                    'modules': ['__init__.py', 'fgm.py']
+                }
+
+        Notes:
+           See https://pages.nist.gov/dioptra/user-guide/api-reference-restapi.html for more
+           information on Dioptra's REST api.
+        """  # noqa B950
+        task_plugin_name_query: str = urljoin(self.task_plugin_builtins_endpoint, name)
+        return cast(dict[str, Any], requests.get(task_plugin_name_query).json())
+
+    def get_custom_task_plugin(self, name: str) -> dict[str, Any]:
+        """Gets a custom task plugin by its unique name.
+
+        Args:
+            name: A unique string identifying a task plugin package within dioptra_builtins collection.
+
+        Returns:
+            The Dioptra REST api's response.
+
+            Example::
+
+                {
+                    'taskPluginName': 'custom_poisoning_plugins',
+                    'collection': 'dioptra_custom',
+                    'modules': ['__init__.py',
+                    'attacks_poison.py',
+                    'data_tensorflow.py',
+                    'datasetup.py',
+                    'defenses_image_preprocessing.py',
+                    'defenses_training.py',
+                    'estimators_keras_classifiers.py',
+                    'registry_art.py',
+                    'tensorflow.py']
+                }
+
+        Notes:
+           See https://pages.nist.gov/dioptra/user-guide/api-reference-restapi.html for more
+           information on Dioptra's REST api.
+        """  # noqa B950
+        task_plugin_name_query: str = urljoin(self.task_plugin_custom_endpoint, name)
+        return cast(dict[str, Any], requests.get(task_plugin_name_query).json())
+
+    def list_experiments(self) -> list[dict[str, Any]]:
+        """Gets a list of all registered experiments.
+
+        Returns:
+            A list of responses detailing all experiments.
+
+            Example::
+
+                [{
+                    'lastModified': '2023-04-24T20:20:27.315687',
+                    'experimentId': 1,
+                    'name': 'mnist',
+                    'createdOn': '2023-04-24T20:20:27.315687'
+                },
+                ...
+                {
+                    'lastModified': '2023-06-22T13:42:35.379462',
+                    'experimentId': 10,
+                    'name': 'mnist_feature_squeezing',
+                    'createdOn': '2023-06-22T13:42:35.379462'
+                }]
```

Comment:
The following whitespace layout would be consistent with the other examples when dealing with list of dicts:

```suggestion
                [
                    {
                        'lastModified': '2023-04-24T20:20:27.315687',
                        'experimentId': 1,
                        'name': 'mnist',
                        'createdOn': '2023-04-24T20:20:27.315687'
                    },
                    ...
                    {
                        'lastModified': '2023-06-22T13:42:35.379462',
                        'experimentId': 10,
                        'name': 'mnist_feature_squeezing',
                        'createdOn': '2023-06-22T13:42:35.379462'
                    }
                ]
```